.travis.yml

```diff
@@ -22,7 +22,7 @@ env:
   - VJS=5
   - VJS=6
 addons:
-  firefox: latest
+  firefox: "54.0"
```

Comment:
This is definitely meant to be temporary until the issue is resolved, I'll add a TODO